MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+# Kubernetes Project Maintainership
+
+Kubernetes is a big project with many contributors, the project organization is
```

Comment:
Can we start this document out with a bit more positive tone?  Yes we have a big task, but it is doable :)